css/styles.css

```diff
@@ -53,7 +53,7 @@ input[type=number]::-webkit-outer-spin-button {
     width: 100%;
     height: 100%;
     margin-top: 50px;
-    display: flex;
+    display: flex; /* Comment */
```

Comment:
You are using flexbox throughout our CSS but you're also setting the width on a bunch of elements.  Flexbox is used so that you don't have to set widths manually on elements.  I'd recommend removing the widths and using [flex-shrink](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-shrink) instead.